manual/doc/extending.tex

```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
+in which the top-level keys declare a new constraint.
+The values in this map are the definitions of a constraint, and are also structured as maps. Valid
+constraint definitions must at least include a \texttt{type} entry that governs the functional
+form of the associated likelihood. \EOS understands the following types of constraints:
+\begin{itemize}[labelwidth=.15\textwidth]
+    \item \texttt{Amoroso},
+    \item \texttt{Gaussian},
+    \item \texttt{MultivariateGaussian},
+    \item \texttt{MultivariateGaussian(Covariance)}.
+\end{itemize}
+
+\subsection{Type \texttt{Amoroso}}
+
+Type \texttt{Amoroso} uses a likelihood arising from the \gls{PDF} of the
+Amoroso distribution \cite{Crooks:2010}, a four-parameter distribution. This
+distribution is useful for the modelling of (sequences) of upper limits on
+as-of-yet undiscovered decays at given probabilities. For example, the upper
+limits on the decay $B_s \to \mu^+\mu^-$ by the LHCb experiment prior to the
+discovery of this decay are modelled using the \texttt{Amoroso} type of
+likelihood. The following keys are required in the description of the
+constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{observable}] the qualified name for an \texttt{Observable} or a \texttt{Parameter};
+    \item[\texttt{kinematics}] the map representation of the kinematic variables pertaining to the observable;
+    \item[\texttt{options}] the map represenation of the options pertaining to the observable;
+    \item[\texttt{physical-limit}] the physical lower bound on the observable;
+    \item[\texttt{theta}] the $\theta$ parameter of the Amoroso distribution;
+    \item[\texttt{alpha}] the $\alpha$ parameter of the Amoroso distribution;
+    \item[\texttt{beta}] the $\beta$ parameter of the Amoroso distribution;
+\end{description}
+We strongly recommend contacting the \EOS authors before adding your own constraint of type \texttt{Amoroso}.
+
+\subsection{Type \texttt{Gaussian}}
+
+Type \texttt{Gaussian} uses a univariate Gaussian or normally-distributed
+likelihood. The following keys are required in the description of the
+constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{observable}] the qualified name of an \texttt{Observable} or a \texttt{Parameter};
+    \item[\texttt{kinematics}] the map representation of the kinematic variables pertaining to the observable;
+    \item[\texttt{options}] the map represenation of the options pertaining to the observable;
+    \item[\texttt{mean}] the mean of the distribution;
+    \item[\texttt{sigma-stat}] the map representation with keys \texttt{hi} and \texttt{lo} for the
+    upper and lower statistical uncertainty of the distribution, respectively;
+    \item[\texttt{sigma-sys}] the same as \texttt{sigma-stat}, but for the systematic uncertainty;
+    \item[\texttt{dof}] the degrees of freedom associated with this observation (should default to $1$).
```

Comment:
Maybe give a comment for what a non-default value could be useful. As far as I remember putting a zero would exclude this constraint from being considered a degree of freedom in the p-value calculation.

Reply:
I think I want to get rid of this anyway. So it makes no sense to document it right now.